contracts/Vault.sol

```diff
@@ -553,7 +555,7 @@ contract Vault is
         if (_strategy == address(0)) revert VaultStrategyNotSet();
         if (IStrategy(_strategy).vault() != address(this))
             revert VaultInvalidVault();
-        if (address(strategy) != address(0) && strategy.hasAssets() == true)
+        if (address(strategy) != address(0) && strategy.hasAssets())
```

Comment:
why this change?

Reply:
it was unnecessary to check if boolean value == true inside an if statement